### PR TITLE
Update jquery.multiselect.filter.css

### PR DESCRIPTION
**Pull request by SteveTheTechie**
Filter text should not be so tiny.

### This Pull Request is a
 - [x] Bug fix
 - [ ] Feature addition
 - [ ] Code refactoring / optimization
 - [ ] Test Addition
 - [ ] Demo Addition
 - [ ] i18n Addition
 - [ ] Other (explain below)
   
### What changes are you proposing?  Why are they needed?<br>
_(Provide use cases, code references, [jsfiddle](https://jsfiddle.net/), [codepen](https://codepen.io), etc.)_   

Filter text should not be so tiny.
Change to .9em's

  
### Pull Request Approval Checklist:
  - [x] No Tabs / Code Spacing Consistent
  - [x] Tests Ran and 0 Failing
  - [x] Tests Added/Updated
  - [x] Impacted Demos Updated
  - [x] Impacted i18n Code Updated
  
### @Mentions:
@mlh758

### CODE REVIEW

**Comment by mlh758** [comment]
I don't think this or the current setting is actually doing anything. They both appear to be getting overridden to 1em by jquery ui.
image

**Comment by SteveTheTechie** [comment]
hmm let me take another look

**Comment by SteveTheTechie** [comment]
@mlh758  I have figured out what is going on here... So let's unpack this.

1.  First, lets observe that the `.ui widget input` and the `.ui-multiselect-filter input` selectors both apply and are actually equal CSS specificity.  Thus, the last rule parsed by the browser is the one that "takes".

2.  In my HTML, I include the jquery-ui css 1st and the multiselect css after that.  I do this because I consider the jquery-ui css to be more generic and I want the multiselect css that I consider to be more specialized to override any rules where appropriate.  This is specifically geared to a case of equal specificity so that the multiselect css will be the winner in the rule comparison.

3.  However, in the filter demo html that I assume you are referencing, the CSS inclusion order is _reversed_.   This means the last rule parsed for this case of equal specificity would be the one from the jquery-ui css not the one from the multiselect css.

In a nutshell. we can certainly change the order, but in reality, what we probably should do is to up the specificity of the `.ui-multiselect-filter input` rule so that it always wins the comparison regardless of order of CSS rules parsed.

2 fixes here then:

1.  Change the order to include jquery-ui css first.  (more for optics though) 
2.  Change the `.ui-multiselect-filter input` selector to be `.ui-widget .ui-multiselect-filter input` in the rule instead.

**Comment by SteveTheTechie** [comment]
I will do another different PR after this one to reorder the CSS loading in the demos so that the jquery-ui css loads first.  It is loading last after the other CSS in all the demos that I see.
